src/Control/Retry.hs

```diff
@@ -1,3 +1,5 @@
+{-# LANGUAGE FlexibleInstances #-}
+{-# LANGUAGE TypeSynonymInstances #-}
```

Comment:
FlexibleInstances, TypeSynonymInstances, FlexibleContexts, MultiParamTypeClasses, and RecordWildCards are unnecessary.